PYME/DSView/DisplayOptionsPanel.py

```diff
@@ -125,7 +125,7 @@ def __init__(self, parent, displayOpts, horizOrientation=False, thresholdControl
             hsizer.Add(self.cbScale, 0, wx.ALL|wx.ALIGN_CENTER, 5)
             vsizer.Add(hsizer, 0, wx.ALL|wx.ALIGN_CENTER, 0)
         else:
-            vsizer.Add(self.bOptimise, 0, wx.LEFT|wx.RIGHT|wx.TOP|wx.ALIGN_CENTER|wx.EXPAND, 5)
```

Comment:
In this case, the `wx.LEFT|wx.RIGHT|wx.TOP` refer to padding (go with the following 5) and should not be removed.